test_regress/t/t_ccache_report.pl

```diff
@@ -12,6 +12,7 @@
 
 if (!$Self->cfg_with_ccache) {
     skip("Requires configuring with ccache");
+    return 1;
```

Comment:
return is exiting a program which will cause a perl warning (when they are enabled), so instead please wrap the rest in an else line that has the other code (or calls a function with the other code), see e.g. t_trace_fst_sc.pl.